### ISSUE TITLE
Importing PyTorch raise a side effect error

### ISSUE THREAD

**Issue by rasenmaeher92** [pschanely/CrossHair#157]
**Expected vs actual behavior**
I was training to analyze some [PyTorch](https://pytorch.org/) related code an unfortunately importing PyTorch fails due to an error about side effects. Unfortunately, I do not know whether there are significant side effects of importing PyTorch, but it would be nice if CrossHair would still work.

```
I couldn't import a file.
/mnt/miniconda3/envs/myenv/lib/python3.9/site-packages/crosshair/util.py:328:
|    try:
|        root_path, module_name = extract_module_from_file(filename)
|        with add_to_pypath(root_path):
>            return import_module(module_name)
|    except Exception as e:
|        raise ErrorDuringImport from e
|

We've blocked a file writing operation on "/dev/null". CrossHair should not be run on code with side effects
-----------------------------------------------------------------------------------------------------------------------
  Analyzed 0 paths in "cross_hair_test.py".       
```

**To Reproduce**
Install PyTorch, e.g. with conda (there are other installation methods listed on the [PyTorch website](https://pytorch.org/)):
```bash
conda install pytorch torchvision torchaudio cudatoolkit=10.2 -c pytorch
```

Create a file `cross_hair_test.py` with the following content:
```python
import torch
```

Run CrossHair:
```bash
python -m crosshair watch cross_hair_test.py
```

**Comment by pschanely** [comment]
This should be fixed with de1f6f3928d5dddaccaf3a18dee3eb9ea5f271dc! I'll update this issue again when I cut a release including it. (this week)

**Comment by pschanely** [comment]
Should be fixed in [v.0.0.22](https://pypi.org/project/crosshair-tool/0.0.22/). Thanks again for the report, and please reopen if it still isn't working for you!
